server/index.js

```diff
@@ -68,7 +67,40 @@ app.get('/comments/:id', async (req, res) => {
   } catch (error) {
     console.error(error);
     return res.status(400).json({
-      succes: false,
+      success: false,
+      msg: error,
+    });
+  }
+});
+
+app.get('/user/comment/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    const comment = await db.getUserComment(id);
+
+    if (!comment || id > 100) {
```

Comment:
When we get to 10M records, it'll probably make sense to put up stronger guardrails here. For now, consider validating `id` before firing off the db call on line 80.

Reply:
I wanted to refactor some of the legacy code after, however added some validations for now.